alphagov/paas-admin
# [#156721553] Add build CI pipeline setup

## What

We're going to use our concourse build CI to tag each merge with a
version number. This is required to keep on consistency across all
environments. The tagging pipeline, also runs integration tests that are
setup on each repo. We're adding that configuration here, but making it
rather noop as we don't currently have integration tests and the unit
tests are run on travis.

## How to review

- Make sure it makes sense
- Make sure it's a noop action

## Dependency

This PR is the first in the chain. Needs to be merged before the:

- alphagov/paas-admin#61
- alphagov/paas-cf#1386
